connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResourceTest.java

```diff
@@ -120,6 +128,31 @@ public ConfigInfos answer() {
         PowerMock.verifyAll();
     }
 
+    @Test
+    public void testListConnectorPlugins() {
+        herder.connectorPlugins();
+
+        PowerMock.expectLastCall().andAnswer(new IAnswer<List<String>>() {
```

Comment:
Since this is basically just a pass-through method anyway, there isn't that much to test here -- you could simplify this test just to use a simple set of connectors you create yourself. There's a lot of code in this test when all you really want to see is that the set makes it back out of the call to ConnectorPluginsResource